src/login/login.jsx

```diff
@@ -25,52 +25,50 @@ const Login = () => {
   };
 
   return (
-    <div>
-      {userLoggedIn && <Navigate to={"/home"} replace={true} />}
-      <main>
+    <div className="loginContainer">
+      <div className="center">
```

Comment:
Does `<div className="loginContainer center">` works?
It's always better to have less divs, so we try minimizing them when possible